src/lib.rs

```diff
@@ -135,7 +135,7 @@ pub use util::amount::Denomination;
 pub use util::amount::SignedAmount;
 pub use util::merkleblock::MerkleBlock;
 pub use util::sighash::SchnorrSigHashType;
-
+pub use util::psbt::{Input, Output, TapTree, PsbtSigHashType};
```

Comment:
I really think exporting type `bitcoin::Input` is highly misleading name, It is for sure should be `bitcoin::psbt::Input` or `bitcoin::util::psbt::Input`, as it always were

Reply:
Oh yes this is much better.